src/libraries/System.Runtime/tests/System/ObjectDisposedExceptionTests.cs

```diff
@@ -42,5 +42,19 @@ public static void Ctor_String_String()
             Assert.Contains(message, exception.Message);
             Assert.Contains(objectName, exception.Message);
         }
+
+        [Fact]
+        public static void Throw_Object()
+        {
+            var obj = new object();
+            AssertExtensions.Throws<ObjectDisposedException>(() => ObjectDisposedException.Throw(obj));
+        }
+
+        [Fact]
+        public static void Throw_Type()
+        {
+            var type = new object().GetType();
```

Comment:
This is an inappropriate use of `var` (it's not receiving the result of a `new` expression or a cast), and a pretty inefficient way of getting a well-known value.

```suggestion
            Type type = typeof(object);
```